src/l500/l500-private.h

```diff
@@ -232,6 +232,22 @@ namespace librealsense
             { L515_PID_PRE_PRQ,     "Intel RealSense L515 (pre-PRQ)"},
             { L515_PID,             "Intel RealSense L515"},
         };
+static std::vector< byte > read_fw_table_raw( hw_monitor & hwm, int table_id)
```

Comment:
How's this different than read_fw_table? If you can't use the other one: please explain, move so that they're next to each other, and perhaps even use one from the other.

Reply:
Done